packages/dds/tree/.mocharc.customBenchmarks.cjs

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Mocha configuration file to run memory-profiling tests
+ */
+"use strict";
+
+const getFluidTestMochaConfig = require("@fluid-internal/mocha-test-setup/mocharc-common");
+
+const packageDir = __dirname;
+const baseConfig = getFluidTestMochaConfig(packageDir);
+
+const nodeOptions =
+	baseConfig["node-option"] !== undefined
+		? Array.isArray(baseConfig["node-option"])
+			? baseConfig["node-option"]
+			: [baseConfig["node-option"]] // If string, wrap as array to use spread operator
+		: []; // If undefined, use an empty array
+
+nodeOptions.push("expose-gc", "gc-global", "unhandled-rejections=strict");
+
+module.exports = {
+	...baseConfig,
+	"fgrep": ["@CustomBenchmark"],
+	"node-option": nodeOptions, // without leading "--"
+	"recursive": true,
+	"reporter": "@fluid-tools/benchmark/dist/MochaReporter.js",
+	"reporterOptions": ["reportDir=.memoryTestsOutput/"],
```

Comment:
We should use a different folder for custom benchmarks. Eventually the pipeline grabs all the files from all the `.memoryTestsOutput/` folders in the repo and processes them as memory-usage benchmarks. In one of the upcoming tasks we'll do something similar for custom benchmarks, but we'll want them to be in their own separate folder.

And together with this, also add this new folder to `.gitignore`.
```suggestion
	"reporterOptions": ["reportDir=.customBenchmarksOutput/"],
```